v1/server/compile_handler_checks_test.go

```diff
@@ -552,6 +552,34 @@ include if user == input.fruits.user`,
 				},
 			},
 		},
+		{ // NOTE(sr): only supported for SQL-ish targets and prisma (see below)
+			note:   "equality with var",
+			rego:   `include if input.fruits.colour = _`,
+			target: "application/vnd.opa.ucast.linq+json",
+			errors: []Error{
+				{
+					Code:    "pe_fragment_error",
+					Message: `existence of field: unsupported feature "existence-ref" for UCAST (LINQ)`,
+				},
+			},
+		},
+		{
+			note:   "equality with var (sql)",
+			rego:   `include if input.fruits.colour = _`,
```

Comment:
Would it be redundant to also assert the "reversed" expression (`_ = input.fruits.colour`)?

Reply:
Yeah, the code doesn't care for 1st or 2nd operand. But I can add one just to be sure.